cli/cmdlineparser.cpp

```diff
@@ -603,25 +603,10 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
             else if (std::strncmp(argv[i], "--platform=", 11) == 0) {
                 const std::string platform(11+argv[i]);
 
-                if (platform == "win32A")
-                    mSettings->platform(Settings::Win32A);
-                else if (platform == "win32W")
-                    mSettings->platform(Settings::Win32W);
-                else if (platform == "win64")
-                    mSettings->platform(Settings::Win64);
-                else if (platform == "unix32")
-                    mSettings->platform(Settings::Unix32);
-                else if (platform == "unix64")
-                    mSettings->platform(Settings::Unix64);
-                else if (platform == "native")
-                    mSettings->platform(Settings::Native);
-                else if (platform == "unspecified")
-                    mSettings->platform(Settings::Unspecified);
-                else if (!mSettings->loadPlatformFile(argv[0], platform, mSettings->verbose)) {
-                    std::string message("unrecognized platform: \"");
-                    message += platform;
-                    message += "\".";
-                    printError(message);
+                std::string errstr;
+                const std::vector<std::string> paths = {argv[0]};
```

Comment:
I think the equal operator is unfortunate.

Reply:
Why? That's how you initialize a container and we do it everywhere.

Reply:
In my opinion it's better with

    const std::vector<std::string> paths{argv[0]};

To be clear I like equal better when RHS is not a c++11 initializer list..

    int a = 4;   // <- I like this more
    int b{5};    // <- than this

Reply:
GCC 4.8 doesn't fully support that syntax so cannot use that.

Reply:
ok understood :-(

Reply:
To be honest I totally forgot about that. Just when I started to change the code it came back to me.